docs/src/extend/firmware-management/building-image/rugpi/tutorial.md

```diff
@@ -0,0 +1,141 @@
+
+    ```sh
+    curl --proto '=https' --tlsv1.2 -sSf https://just.systems/install.sh | sudo bash -s -- --to /usr/bin
+    ```
+
+    Check out the [justfile website](https://just.systems/man/en/chapter_5.html) for more installation options.
+
+### Customizing your own image
+
+1. Fork the project
+
+2. Edit the authorized ssh keys for each profile
+
+    Edit the `root_authorized_keys` under each toml file in the `profiles/` directory, and add the public ssh keys whom you want to grant ssh access to for the device. Below shows one example.
+
+    ```toml title="file: profiles/default.toml"
+    #...
+
+    [parameters.ssh]
+    root_authorized_keys = """
+    ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDfhQGWWw73ponAokdNSRZ5cQc9/CIX1TLQgYlr+BtObKoO4UNFP1YSbgK03GjhjeUid+QPmV+UURqxQTqLQoYWqUFP2CYkILFccVPmTvx9HLwupI+6QQKWfMDx9Djfph9GzInymaA5fT7hKppqittFrC/l3lkKgKTX5ohEOGshIbRgtgOYIaW3ByTx3urnaBbYCIgOyOZzSIyS0dUkwsiLu3XjPspgmn3Fs/+vofT/yhBe1carW0UM3ivV0JFfJzrxbCl/F7I2qwfjZXsypjkwlpNupUMuo3xPMi8YvNvyEu4d+IEAqO1dCcdGcxlkiHxrdITIpVLt5mjJ2LauHE/H bootstrap
+    """
+
+    # ...
+    ```
+
+3. Commit the changes
+
+4. You can now add any custom recipes to your project if you want to include additional packages in your image, or configure items.
+
+    See the [Rugpi System Customization](https://oss.silitics.com/rugpi/docs/guide/system-customization) docs for details on how to do this or follow clues from some of the existing recipes included in the [tedge-rugpi-images](https://github.com/thin-edge/tedge-rugpi-image) project.
+
+### Building an image
+
+:::info
+Currently building is only supported on a Linux environment. It is strongly encouraged to use the Github workflow to build the image.
+:::
+
+1. Build an image for your device (machine)
+
+    ```sh tab={"label":"Pi\tZero"}
+    just IMAGE_ARCH=armhf PROFILE=armhf VARIANT=pi01 build-all
+    ```
+
+    ```sh tab={"label":"Pi\t1"}
+    just IMAGE_ARCH=armhf PROFILE=armhf VARIANT=pi01 build-all
+    ```    
+
+    ```sh tab={"label":"Pi\t2"}
+    just IMAGE_ARCH=arm64 PROFILE=default VARIANT=pi023 build-all
+    ```
+    ```sh tab={"label":"Pi\t3"}
+    just IMAGE_ARCH=arm64 PROFILE=default VARIANT=pi023 build-all
+    ```
+    ```sh tab={"label":"Pi\tZero2W"}
+    just IMAGE_ARCH=arm64 PROFILE=default VARIANT=pi023 build-all
+    ```
+
+    ```sh tab={"label":"Pi\t4\t(With\tFirmware)"}
+    just IMAGE_ARCH=arm64 PROFILE=default VARIANT=pi4 build-all
+    ```
+
+    ```sh tab={"label":"Pi\t4\t(Without\tFirmware)"}
+    just IMAGE_ARCH=arm64 PROFILE=default VARIANT=pi45 build-all
+    ```
+
+    ```sh tab={"label":"Pi\t5"}
+    just IMAGE_ARCH=arm64 PROFILE=default VARIANT=pi45 build-all
+    ```
+
+2. Inspect the built image
+
+    ```sh
+    ls -ltr build/*.img
+    ```
+
+3. Flash the base image using the instructions on the [Flashing an image](../../flashing-an-image) page
+
+
+## Tips
```

Comment:
A tip that could be useful is how to choose the appropriate image.

I have a Pi400, do I have to pick a Pi4 with or without firmware? something else?

Reply:
Yeah the part about the with or without the firmware is unfortunately more complex then I'd hoped...but it is mostly due to the fact that the "tryboot" feature was only introduced after pi4 was released :/ So I definitely need to expand on that.

Reply:
I've added a note which references the tip section. c134c874fd473c49cbb565e2b85310b56ab9a09a